doc/modules/changes/20230712_ranpengli

```diff
@@ -0,0 +1,4 @@
+Changed: The entropy plugin now includes a plasticity rheology, which can be set by cohesion and frictional angle. It can also use a viscosity prefactor profile, set the lateral viscosity variation limit, and choose p-T dependent conductivity
+
+<br>
+(Ranpeng Li, 2023/07/12)
```

Comment:
You probably want to include Juliane as author here as well, or did you do all of these changes on your own?